Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -542,17 +543,26 @@ def _is_legit_currency(currency):
 def get_all_symbols_dict():
     global all_symbols_dict
     if not all_symbols_dict:
+        request_response = None
         try:
+            # inspired from https://github.com/man-c/pycoingecko
+            session = requests.Session()
+            retries = requests.packages.urllib3.util.retry.Retry(total=5, backoff_factor=0.5,
```

Comment:
Interesting method, does an equivalent exists in aiohttp ?

Reply:
Probably, here we don't need it async though, it's done on purpose when the web interface needs it